config/locales/views/rents/rents.sv.yml

```diff
@@ -1,62 +1,57 @@
+---
 sv:
   rents:
     form:
-      allowed_to_overbook: kan boka över en bokning som börjar mer än 5 dagar fram i tiden
-      cannot_edit_rent: Du kan inte längre redigera bokningen
+      the_terms: bokningsavtalet.
+      read_terms: "Läs igenom och godkänn"
+      agree_to_terms: "Jag godkänner bokningsavtalet"
+      school_hours_warning: "Bokning med start- eller sluttid utanför skoltimmar (vardagar 8-17) måste godkännas manuellt"
+      note: OBS!
+      allowed_to_overbook: "kan boka över en bokning som börjar mer än 5 dagar fram i tiden"
+      cannot_edit_rent: "Du kan inte längre redigera bokningen"
       council_booking: Utskottsbokning
-      edit_user_information: Redigera användaruppgifter
-      only_for_council: får endast göras åt utskottet
-      overlaps_with: 'Överlappar med:'
+      edit_user_information: "Redigera användaruppgifter"
```

Comment:
Quotes are not needed in yml:s
And if we wanted them, you have mixed single and double quotes